signing/secp256k1/impl/src/test/java/tech/pegasys/signers/secp256k1/multikey/MultiKeySignerProviderTest.java

```diff
@@ -132,4 +132,22 @@ void signerIsLoadedSuccessfullyWhenAddressHasCaseMismatchToFilename() throws URI
     assertThat(signer).isNotNull();
     assertThat(signer.getPublicKey().toString()).isEqualTo("0x" + LOWER_CASE_PUBLIC_KEY);
   }
+
+  @Test
+  void signerIsNotReturnedIfFileContainsPublicKeyDifferentToRequest() throws URISyntaxException {
+
+    final FileBasedSigningMetadataFile capitalisedMetadata =
+        new FileBasedSigningMetadataFile(
+            LOWERCASE_ADDRESS.toUpperCase() + ".toml",
```

Comment:
Does the address need to be uppercase? Didn't think this would matter for this test

Reply:
correct, sign is irrelevant for this test.